src/seedu/addressbook/commands/EmailSortCommand.java

```diff
@@ -0,0 +1,27 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.Person;
+
+import java.util.Comparator;
+import java.util.List;
+
+/**
+ * Sorts all Email address of persons in the address book to the user.
+ */
+public class EmailSortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sortemail";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Sorts all persons's email address in the address book alphabetically and display them as a list with index numbers.\n"
```

Comment:
Avoid lines longer than 110 chars. When a line is too long, it requires horizontal scrolling, that slows down the reader/reviewer.